### ISSUE TITLE
Cannot the Simple Calendar module

### ISSUE THREAD

**Issue by ThwillArtz** [Pokerole-Software-Development/foundry-pokerole#53]
**Describe the bug**
I can't open the simple calendar at all. Other functions such as using it as a dependency for other modules like Small weather or Weather Control appear to work fine.
While I understand that the system is very much in its early stages. I figured it was a problem worth looking into as it might snowball into other things.

**To Reproduce**

1.     Create a Pokérole world in Foundry
2.     Going to settings or Double click the SC button under the journal quick bar
3.     Opening the calendar or Attempting to edit settings
4.     See error

<img width="274" alt="image" src="https://user-images.githubusercontent.com/118507524/228172667-1766219c-f793-40f8-8590-2d32f0759342.png">
<img width="481" alt="image" src="https://user-images.githubusercontent.com/118507524/228171990-cf3a37bc-6d11-405a-937c-53550f3d826c.png">
<img width="373" alt="image" src="https://user-images.githubusercontent.com/118507524/228172304-e974ff8f-62ee-49fd-9afa-95dfcd99159e.png">

```
 static onError(location, error, {msg="", notify=null, log=null, ...data}={}) {
    if ( !(error instanceof Error) ) return;
    if ( msg ) error.message = `${msg}. ${error.message}`;
    if ( log ) console[log]?.(error);
    if ( notify ) ui.notifications[notify]?.(msg || error.message);
```

**Comment by tech-ticks** [comment]
Reopened because there still seems to be an error when opening calendar settings

**Comment by tech-ticks** [comment]
Fixed in v0.3.1
